framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -323,6 +323,10 @@
     // org.checkerframework.framework.util.typeinference.DefaultTypeArgumentInference
     "showInferenceSteps",
 
+    // Output stack trace when checker encounters errors
+    // org.checkerframework.common.basetype.BaseTypeChecker.printStackTrace()
```

Comment:
Shouldn't the option be handled in this class, in method `printOrStoreMessage`, too?

Reply:
I thought we had to mention the place where the dumpOnErrors option was being checked. Since I was checking for the option in the printStackTrace() method, I wrote its path. The option is not used at all in the printOrStoreMessage (only the printStackTrace() method is called). Currently, I have not made any changes. Please let me know what you think I should do.